libs/bunny-storage/package.json

```
@@ -2,12 +2,22 @@
   "name": "@bunny.net/storage-sdk",
   "version": "0.2.2",
   "main": "dist/lib.js",
-  "module": "esm/lib.mjs",
+  "module": "esm-node/lib.mjs",
   "types": "dist/_tsup-dts-rollup.d.ts",
+  "exports": {
+    ".": {
+      "import": "./esm/lib.mjs",
+      "require": "./dist/lib.js",
+      "types": "./dist/_tsup-dts-rollup.d.ts",
+      "deno": "./esm/lib.mjs",
+      "node": "./esm-node/lib.mjs"
+    }
+  },
```

Hello, I checked out this branch locally and built it to my project's local `node_modules/`. However it looks like my Node.js v23.10.0 prefers to import from `node_modules/@bunny.net/edgescript-sdk/esm/lib.mjs` instead of `node_modules/@bunny.net/edgescript-sdk/node-esm/lib.mjs`, so the issue remains.

In the Node.js documentation on [conditional exports](https://nodejs.org/api/packages.html#conditional-exports), it says:

> Within the ["exports"](https://nodejs.org/api/packages.html#exports) object, key order is significant. During condition matching, earlier entries have higher priority and take precedence over later entries. The general rule is that conditions should be from most specific to least specific in object order.

And indeed, moving the `node` key above the `import` seems to fix the issue!

```suggestion
  "exports": {
    ".": {
      "deno": "./esm/lib.mjs",
      "node": "./esm-node/lib.mjs",
      "import": "./esm/lib.mjs",
      "require": "./dist/lib.js",
      "types": "./dist/_tsup-dts-rollup.d.ts"
    }
  },
```